### PR TITLE
move addObject to helpers, remove unneeded containers, change type in…

### PR DESCRIPTION
**Pull request by kellymiller6**
## What's this PR do?
Move addObject function out of actions and into helper, get rid of unused containers, change input types to accurately reflect data, style and label inputs
## Pre-Merge TODOs
 - [x] Screenshots Included
    - ANY change to the UI of the app must have screenshots/gifs
    - Hint: recordit is really great for this
 - [ ] 2 PR reviews completed

## Learning

My addObject action really had nothing to do with redux, so I moved it out of actions and into a helper file. I was able to get rid of containers for addSitter and addWorker, I decided to keep the container for addChild because of functionality I plan on implementing in the future. 

I also changed my input 'types' to more accurately reflect the data. I added labels and got rid of placeholders where it made sense to me. 

## Where should the reviewer start?

actions.js, helper.js, the onClick for the button on the bottom of all my 'add' files.
the input types in all three 'add files'

## Screenshots (if appropriate)
screen shot 2017-06-06 at 11 47 25 am

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -1,26 +1,5 @@
 var CryptoJS = require("crypto-js")
 import { firebaseApp, ref} from '../firebase/Firebase';
```
Comment (Adamj1232) [inline src/actions/index.js:2]: Extra space at beginning of import inside curlies

File: src/containers/AddChildContainer.js [hunk 2]
```diff
@@ -9,10 +8,10 @@ const mapStateToProps = (state) => {
   }
 }
 
-const mapDispatchToProps = (dispatch) => {
-  return {
-    addChild: (child) => dispatch(addObject(child, 'children'))
-  }
-}
+// const mapDispatchToProps = (dispatch) => {
```
Comment (Adamj1232) [inline src/containers/AddChildContainer.js:11]: ditch the comments?

File: src/actions/index.js (hunk 1 above)
Comment (kellymiller6) [reply src/actions/index.js:2]: thanks!

**Comment by Adamj1232** [comment]
Looks good, a few extra spaces on one side of curly brackets and a few commented out sections that can prob go but it looks great!!

File: src/components/Children/AddChild.js [hunk 3]
```diff
@@ -19,57 +19,59 @@ export default class AddChild extends Component {
...
                   })}
           />
+          <label>Worker Number:</label>
           <input className='add-input'
-                  type='text'
-                  value={this.state.childInsurance}
-                  placeholder='Worker Number'
+                  type='tel'
+                  value={this.state.Worker_Number}
+                  placeholder='xxx-xxx-xxxx'
```
Comment (letakeane) [inline src/components/Children/AddChild.js:66]: For a future feature, might be nice to see the phone number portion of the form get some logic so the user doesn't have to put in the "-"s? Unless that's built into the 'tel' type?

**Review by letakeane — COMMENTED** [review]
Nice work refactoring and ditching the extraneous Containers :)

File: src/styles/app.css [hunk 4]
```diff
@@ -53,11 +71,17 @@ h4{
   font-family: 'Nunito', sans-serif;
 }
 
+label{
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px;
+  color: #185253;
+ }
 input{
```
Comment (JohnBinning) [inline src/styles/app.css:79]: Consider adding an empty line between 78 and 79.  

Also be consistent with the spacing here:

Line 79 - input{
Line 87 - .nav-btn {

I had to look pretty hard to find stuff, good work

**Pull request merged**